point-addition/src/conversion.rs

```diff
@@ -0,0 +1,91 @@
+//! This module implements the conversion from the sum of two elliptic curve points to the sum of
+//! two field elements of the field underlying the elliptic curve
```

Comment:
I suggest to also prime the reader (or potential contributors who try to understand our code now) by adding

//! The protocol is described in https://docs.tlsnotary.org/protocol/notarization/key_exchange.html

(at least temporarily, we will later have a dedicated point_addition page and will update this comment)